qa/ccs-common-rest/build.gradle

```diff
@@ -39,7 +39,9 @@ tasks.named("yamlRestTest") {
       'search.aggregation/50_filter/Standard queries get cached',
       'search.aggregation/50_filter/Terms lookup gets cached', // terms lookup by "index" doesn't seem to work correctly
       'search.aggregation/70_adjacency_matrix/Terms lookup', // terms lookup by "index" doesn't seem to work correctly
-      'async_search/20-with-poin-in-time/Async search with point in time' // [indices] cannot be used with point in time
+      'async_search/20-with-poin-in-time/Async search with point in time', // [indices] cannot be used with point in time
+      'search.vectors/40_knn_search/Knn vector query with stored vector', // query_vector lookup by "index" doesn't seem to work correctly
+      'search.vectors/40_knn_search/Knn vector query with missing stored vector' // query_vector lookup by "index" doesn't seem to work correctly
```

Comment:
@jdconrad it has to do with "Write indices" being on the remote cluster and "search" indices being on the local cluster. I *think* it is suppose to add the remove cluster prefix automatically for certain things, but I bet it is not smart enough to do this for fetching a document within the query. 

Terms fetching has the same issue (search earlier in the block list).

Reply:
this is because both rely on get that does not support cross-cluster execution? The doc to be fetched should then be on the cluster where the search is being coordinated?